html-css/lesson1.md

```diff
@@ -44,15 +46,18 @@ We'll cover the following semantic tags:
   - ``<article>``
   - ``<aside role="complementary">``
 
-> **Exercise (pair programming):**  Work in pairs to determine where to place these new Tags and Attributes in the index.html file. Who benefits when we write "semantic" HTML?
+> **Exercise (pair programming):**  Switch between the person who is driving and the one who is helping out. The person who is now driving will `fork` the repository and `clone` it to their computer. Add the gray section with the text 'Bikes for Refugees' and the two buttons. Update the website to use semantic tags.
 
 ## CSS Selectors
 
 During your application process, you became familiar with CSS selectors. We'll review the basic selectors and then practice combining these to modify our button styles.
 
 > If you want to review the selectors, read the [Common Selectors section](http://learn.shayhowe.com/advanced-html-css/complex-selectors/) of this page.
 
-> **Exercise (pair programming):** Work in pairs to make the blue buttons on the page red (`#ce5f31`). The white button, which says "Volunteer", should remain white but the text should change to red.
+> **Exercise (pair programming):** Continue working in pairs and use the [background properties](http://www.htmldog.com/references/css/properties/background/) to add a [assets/jumbotron-background.jpg](background image) to the jumbotron and change the colour of the text to white.
```

Comment:
[assets/jumbotron-background.jpg](background image) shows up as unformatted. I added an `images/header-bike.jpg` to the `bikes-for-refugees` repo, so maybe we can update this to something like "in the bikes-for-refugees repository, find `images/header-bike.jpg` and add it as a background image to the header?